services/user.py

```diff
@@ -0,0 +1,54 @@
+from db.models import User
+from django.core.exceptions import ValidationError
+
+
+def create_user(
+        username: str,
+        password: str,
+        email: str = None,
+        first_name: str = None,
+        last_name: str = None
+) -> list:
+    try:
+        user = User.objects.create_user(
```

Comment:
Use `get_user_model()` instead of `User` here and in the other places as mentioned in the checklist

Reply:
> not fixed

I didn't understand it completely. I defined User as get_user_model(), what's the mistake? How to do it correctly?